src/System.IdentityModel.Tokens/Utility.cs

```diff
@@ -65,5 +65,25 @@ internal static string SerializeAsSingleCommaDelimitedString(IEnumerable<string>
 
             return sb.ToString();
         }
+
+        public static bool IsHttps(string address)
+        {
+            if (string.IsNullOrEmpty(address))
+            {
+                return false;
+            }
+
+            return address.StartsWith("https");
+        }
+
+        public static bool IsHttps(Uri uri)
+        {
+            if (uri == null)
+            {
+                return false;
+            }
+
+            return uri.Scheme == "https";
```

Comment:
The reason I did harcoded string comparison is because Uri.UriSchemeHttps is not available on DNXcore